source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "WeatherFlow"
+description: "WeatherFlow weather sensor"
+ha_release: "0.38"
+ha_category: 
+  - Sensor
```

Comment:
Should "Environment" be added here too?

Reply:
All of the items in the source PR are in `sensor.py` -  I can't seem to find the mention of `Environment` in either the HA docs or the developer docs ... but I'm happy to add it.

Reply:
These are the categories the integration is listed on the website...

Environment:

https://www.home-assistant.io/integrations/#environment

../Frenck

Reply:
Oh that's a hard one -> its a weather station... so maybe it should be under weather -> but we aren't doing any forecasting so maybe environment... 🤔️